src/LoginPage.cc

```diff
@@ -142,63 +177,111 @@ void LoginPage::loginError(QString error)
 	error_label_->setText(error);
 }
 
-void LoginPage::onLoginButtonClicked()
+void LoginPage::onMatrixIdEntered()
 {
+	guessing_server = true;
```

Comment:
You're right, I started with snake case for member variables but camel case seems to be a 
 better option for longer names.